lib/x/client_credentials.rb

```diff
@@ -32,6 +32,24 @@ module ClientCredentials
     # @example Get the bearer token
     #   client.bearer_token
     attr_reader :bearer_token
+    # The OAuth 2.0 client ID
+    # @api public
+    # @return [String, nil] the OAuth 2.0 client ID
+    # @example Get the client ID
+    #   client.client_id
+    attr_reader :client_id
+    # The OAuth 2.0 client secret
+    # @api public
+    # @return [String, nil] the OAuth 2.0 client secret
+    # @example Get the client secret
+    #   client.client_secret
+    attr_reader :client_secret
+    # The OAuth 2.0 refresh token
+    # @api public
+    # @return [String, nil] the OAuth 2.0 refresh token
+    # @example Get the refresh token
+    #   client.refresh_token
+    attr_reader :refresh_token
```

Comment:
You don't want to define all attributes as `attr_reader :client_id, :client_secret, :refresh_token` ?

Reply:
These comments produce YARD documentation that looks like this. I've added 100% documentation coverage as a requirement to this project.

<img width="1624" height="1061" alt="Image" src="https://github.com/user-attachments/assets/0a23c8bf-9043-40b9-8be5-899d15b0c1e2" />

Reply:
Ok, thanks. Can you just add a line break between each attribute ?